docs/src/pages/components/data-grid/events/CatalogOfEventsNoSnap.js

```diff
@@ -10,33 +22,112 @@ const useStyles = makeStyles({
   },
 });
 
+const EventRow = ({ event }) => {
+  const [open, setOpen] = React.useState(false);
+
+  const example = React.useMemo(() => {
+    const args = ['details // GridCallbackDetails'];
+    if (event.event) {
+      args.unshift(`event,  // ${event.event}`);
+    }
+    if (event.params) {
+      args.unshift(`params, // ${event.params}`);
+    }
+
+    return `
+const onEvent: GridEventListener<GridEvents.${event.name}> =
+ (
+    ${args.join('\n    ')}
+  ) => {...}    
+  
+// Imperative subscription    
+apiRef.current.subscribeEvent(
+  GridEvents.${event.name},
+  onEvent,
+);
+
+// Hook subscription (only available inside the Grid scope)
+useGridApiEventHandler(GridEvents.${event.name}, onEvent);    
+`;
+  }, [event]);
+
+  return (
+    <React.Fragment>
+      <TableRow key={event.name}>
+        <TableCell>
+          <IconButton
+            aria-label="expand row"
+            size="small"
+            onClick={() => setOpen(!open)}
+          >
+            {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+          </IconButton>
+        </TableCell>
+        <TableCell>
+          <code>{event.name}</code>
+        </TableCell>
+        <TableCell>
+          <div
+            dangerouslySetInnerHTML={{
+              __html: event.description,
+            }}
+          />
+          {!!event.params && (
+            <div>
+              <Typography variant="subtitle2" component="span">
+                Params:{' '}
+              </Typography>
+              <span
+                dangerouslySetInnerHTML={{
+                  __html: event.params,
+                }}
+              />
+            </div>
+          )}
+          {!!event.event && (
+            <div>
+              <Typography variant="subtitle2" component="span">
+                Event:{' '}
+              </Typography>
+              <span
+                dangerouslySetInnerHTML={{
+                  __html: event.event,
+                }}
+              />
+            </div>
+          )}
+        </TableCell>
+      </TableRow>
+      <TableRow>
```

Comment:
Always rendering the child causes a double-border:

<img width="400" src="https://user-images.githubusercontent.com/42154031/141830165-8e0e50f3-0ab2-450c-85ca-c713fa78d7ac.png" alt="image" style="max-width: 100%;">

Reply:
I followed the demo on the Core documentation, I'll see how to improve it

Reply:
Fixed